### ISSUE TITLE
Orama build generates wrong path in stemmers.d.ts

### ISSUE THREAD

**Issue by toriphes** [oramasearch/orama#361]
**Describe the bug**
I was trying to use orama 1.0.0-beta.12 on an angular project, when I try to import the `create` component from the library I get several typescript errors like:

```
node_modules/@orama/orama/dist/components/tokenizer/stemmers.d.ts:1:35 - error TS2307: Cannot find module '@stemmers/ar.js' or its corresponding type declarations.

1 import { stemmer as arabic } from '@stemmers/ar.js';
[...]
```

I tried to figure out what the problem might be in beta.12 and I'm pretty sure the cause is in the Orama build script:
The script starts generating the ESM package and the `stemmers.d.ts` file just generated is also modified to fix the importing path `@stemmers` with `../../stemmers`. 
Then the CJS package is generated, the typescript compiler is invoked again replacing the previously fixed `stemmers.d.ts` file, so the paths in the file are in the `@stemmers` version again.

I think the solution is to move the "path fixing procedure" to a later step after the generation of EMS and CJS packages.

If you think this could be a good solution, I'd be very happy to create a PR on this.

Thank you for creating OramaSearch!

**To Reproduce**
1. Create and Angular 15.* project
2. `npm i @orama/orama`
3. Add  `import { create } from '@orama/orama'` inside `app.components.ts` file and save

**Additional context**
Searching through the issues I saw that this error had happened already reported in #322 and was fixed in version 1.0.0-beta.6 which works as expected also in my angular project.

**Comment by micheleriva** [comment]
cc. @ShogunPanda

**Comment by ShogunPanda** [comment]
This will be solved by #376.

**Comment by ShogunPanda** [comment]
@toriphes This should have been fixed in beta 16. I'm closing this issue for now. If it's not the case please let us and we will take a look!
